pkg/webhook/webhook.go

```diff
@@ -175,3 +192,102 @@ func (c *Webhook) validateHandlerFunc(review *admission_v1beta1.AdmissionReview)
 		Allowed: true,
 	}
 }
+
+func (c *Webhook) validateRelease(request *admission_v1beta1.AdmissionRequest, release shipper.Release) error {
+	var err error
+	overrideRBs := rolloutblock.NewOverride(release.Annotations[shipper.RolloutBlocksOverrideAnnotation])
+	err = c.validateOverrideRolloutBlockAnnotation(overrideRBs, release.Namespace)
+	if err != nil {
+		return err
+	}
+
+	if request.Operation == kubeclient.Create {
+		err = c.processRolloutBlocks(release.Namespace, overrideRBs)
+	}
+
+	if request.Operation == kubeclient.Update {
```

Comment:
These two if statements look mutually exclusive. I'd prefer either a switch, or an if-else-if, for the sake of explicitness of intent.